databases/week1/week1.sql

```diff
@@ -0,0 +1,48 @@
+show DATABASES;
+use my_database_week1;
+SELECT * 
+from task
+SELECT COUNT (id) from task
+SELECT COUNT(id) from task
+ where due_date is NULL
+
+
+SELECT task.id, title, description, created, updated,due_date, user_id, status.name
+from task
+JOIN status ON task.status_id = status.id 
+where status.name= 'Done' 
+
+SELECT task.id, title, description, created, updated,due_date, user_id, status.name
+from task
+JOIN status ON task.status_id = status.id 
+where status.name != 'Done' 
+
+SELECT * from task
+ORDER BY(created)
```

Comment:
By default, MySQL sorts values in **ascending** order, not descending. This means that this query sorts `created` dates by the smallest first, and the smallest date is the earliets, meaning **least recent**, not most recent.